miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/BoxableDto.java

```diff
@@ -0,0 +1,79 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+public class BoxableDto {
+  private String alias;
+  private String boxAlias;
+  private String boxPosition;
+
+  private Boolean empty;
+
+  private Long id;
```

Comment:
This could easily be confused with the actual ID of the item -- change this to `boxPositionId`?